redis/5.2/modules/ROOT/pages/redis-connector-examples.adoc

```diff
@@ -0,0 +1,282 @@
+        http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+        http://www.mulesoft.org/schema/mule/http
+        http://www.mulesoft.org/schema/mule/http/current/mule-http.xsd
+        http://www.mulesoft.org/schema/mule/ee/tracking
+        http://www.mulesoft.org/schema/mule/ee/tracking/current/mule-tracking-ee.xsd
+        http://www.mulesoft.org/schema/mule/redis
+        http://www.mulesoft.org/schema/mule/redis/current/mule-redis.xsd">
+    <redis:config
+            name="Redis__Configuration"
+            host="${config.host}"
+            connectionTimeout="${config.connectionTimeout}"
+            port="${config.port}"
+            doc:name="Redis: Configuration"/>
+​
+    <http:listener-config name="HTTP_Listener_config" doc:name="HTTP Listener config" basePath="/">
+		<http:listener-connection host="0.0.0.0" port="8081" />
+	</http:listener-config>
+	<flow name="set-flow">
+        <http:listener
+                config-ref="HTTP_Listener_config"
+                path="/"
+                doc:name="Set value HTTP endpoint"/>
+        <redis:set
+                config-ref="Redis__Configuration"
+                key="#[payload.key]"
+                value="#[payload.value]"
+        doc:name="Set value for key into Redis">
+			<redis:value ><![CDATA[payload.key]]></redis:value>
+		</redis:set>
+        <set-payload
+         value="Successfully set value: #[payload.value] to key: #[payload.key]"
+         doc:name="Set value response"/>
+    </flow>
+</mule>
+----
+
+[[xmlobjectstore]]
+
+== Use Redis Connector as a Custom Object Store
+
+To use XML to configure Redis Connector as a custom object store in a Mule app:
+
+. Add the Redis namespace to the `<mule>` element:
++
+[source]
+----
+xmlns:redis="http://www.mulesoft.org/schema/mule/redis"
+----
++
+. Add the location of the Redis schema referred to by the Redis namespace:
++
+[source,text,linenums]
+----
+http://www.mulesoft.org/schema/mule/redis
+http://www.mulesoft.org/schema/mule/redis/current/mule-redis.xs
+----
++
+. Add the HTTP namespace to the `<mule>` element:
++
+[source]
+----
+xmlns:http="http://www.mulesoft.org/schema/mule/http"
+----
++
+. Add the location of the HTTP schema referred to by the HTTP namespace:
++
+[source,text,linenums]
+----
+http://www.mulesoft.org/schema/mule/http
+http://www.mulesoft.org/schema/mule/http/current/mule-http.xsd
+----
++
+. Add the object store namespace to the `<mule>` element:
++
+[source]
+----
+xmlns:os="http://www.mulesoft.org/schema/mule/os"
+----
++
+. Add the location of the object store schema referred to by the object store's namespace:
++
+[source,text,linenums]
+----
+http://www.mulesoft.org/schema/mule/os
+http://www.mulesoft.org/schema/mule/os/current/mule-os.xsd
+----
++
+. Add an `<os:config>` element to your project and configure it to contain a Redis connection type (`clustered-connection` or `nonclustered-connection`). You must use the Studio XML editor to do this.
```

Comment:
```suggestion
. Add an `<os:config>` element to your project and configure it to contain a Redis connection type (`clustered-connection` or `nonclustered-connection`).
```

Reply:
I would remove that phrase and add it at the beginning of the topic since you are configuring all the connector in the Studio XML editor.

Reply:
A user who reads this could be using another XML editor and could be looking at this section to learn. I think it's important to leave this in.